os/include/tinyara/serial/serial.h

```diff
@@ -132,6 +132,16 @@
  * Public Types
  ************************************************************************************/
 
+#ifdef CONFIG_DYNAMIC_LOGS_STORING
+struct log_net_chunk_s {
```

Comment:
I think this structure will not be shared to other module, only used in serial module.
So It should moved to serial.c